docs/lumi.md

```diff
@@ -45,9 +45,9 @@ Loading `helmi_qiskit` or `helmi_cirq` on LUMI comes with a preconfigured Python
 
 | Package                                             | Version        |
 | --------------------------------------------------- | -------------- |
-| [iqm-client ](https://pypi.org/project/iqm-client/) | >=15.2 < 16.0  |
-| [qiskit-iqm](https://pypi.org/project/qiskit-iqm/)  | >=11.10 < 12.0 |
-| [cirq-iqm](https://pypi.org/project/qiskit-iqm/)    | >=12.2 < 13.0  |
+| [iqm-client ](https://pypi.org/project/iqm-client/) | >=18.0 < 19.0  |
+| [qiskit-iqm](https://pypi.org/project/qiskit-iqm/)  | >=13.15 < 14.0 |
```

Comment:
Those should be updated to the latest versions

Reply:
fixed in latest commit